src/amo/components/StaticPages/ReviewGuide.js

```diff
@@ -0,0 +1,117 @@
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { compose } from 'redux';
+
+import Card from 'ui/components/Card';
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+
+import 'amo/components/StaticPages/StaticPages.scss';
+
+
+export class ReviewGuideBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { i18n } = this.props;
+
+    return (
+
+      <Card
+        className="StaticPage"
+        header={i18n.gettext('Review Guidelines')}
+      >
+        <Helmet>
+          <title>{i18n.gettext('Review Guidelines')}</title>
+        </Helmet>
+
+        <section id="review-guide">
+          <p>
+            {i18n.gettext(`Add-on reviews are a way for you to share your opinions
+              about the add-ons you’ve installed and used. Our review moderation
+              team reserves the right to refuse or remove any review that does not
+              comply with these guidelines.`)}
+          </p>
+          <h2>{i18n.gettext('Some tips for writing a great review')}</h2>
+          <h3>{i18n.gettext('Do:')}</h3>
```

Comment:
Urgh, I really don't like these "Do/Don't" styles of pages... it's not very clear and quite infantilizing.

I have a better idea for reorganising. Remove this header... I'll make a suggestion for the "Don't" one.